pandokia/env_platforms.py

```diff
@@ -22,13 +27,22 @@
                i686='x86')
 
 
-class PlatformType(object):
+class 
```

Comment:
I find this spacing odd (why not `class Type(object):` ?), but OK

Reply:
Heh, I *wondered* about that! Did not realize I had hit an extra carriage return while editing. Thanks.